app/scss/partials/_summary.scss

```diff
@@ -11,306 +11,205 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0
  */
 
-#content-summary {
+.Summary {
```

Comment:
Looks good.  I may take a 3rd pass but it looks clean and understandable. Go BEM!

Reply:
Cool beans ~